.github/workflows/build.yml

```diff
@@ -0,0 +1,12 @@
+name: build
+
+on:
+  push:
+    branches:
+    - main
+  pull_request:
+
+jobs:
+  build:
+    uses: jbanghub/.github/.github/workflows/shared-build.yml@main
+    secrets: inherit
```

Comment:
BTW, this has no effect as you can share only secrets in reusable workflows within the same org or between org in the same enterprise. 

Beside if that it looks also pretty dangerous 

See https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#jobsjob_idsecretsinherit

Reply:
good point - not actually sure why that is there by default. but i'll remove the .github part anyway